workshops/models.py

```diff
@@ -404,6 +404,14 @@ def clean(self):
         self.email = self.email.lower() if self.email else None
 
     def save(self, *args, **kwargs):
+        # If GitHub username has changed, clear UserSocialAuth table for this
+        # person.
+        if self.pk is not None:
+            orig = Person.objects.get(pk=self.pk)
+            github_username_has_changed = orig.github != self.github
+            if github_username_has_changed:
```

Comment:
The name of this variable explains what we are actually testing. This could be rewritten as:
```python
if orig.github != self.github:  # if github username has changed
```